source/users/enable-user-metadata.txt

```diff
@@ -226,13 +226,10 @@ Procedure
 Access User Metadata from a Client Application
```

Comment:
A couple blank lines above this heading  - are they there on purpose?

Reply:
Not that I know of! I'll clean that up.

Reply:
Now that I'm looking, there are extra lines all over in this file...